libs/common/src/config/logging_builder.cpp

```diff
@@ -60,7 +60,9 @@ LoggingBuilder::BasicLogging& LoggingBuilder::BasicLogging::Tag(
     return *this;
 }
 LoggingBuilder::BasicLogging::BasicLogging()
-    : level_(Defaults<AnySDK>::LogLevel()), tag_(Defaults<AnySDK>::LogTag()) {}
+    : level_(GetLogLevelEnum(std::getenv("LD_LOG_LEVEL"),
+                             Defaults<AnySDK>::LogLevel())),
+      tag_(Defaults<AnySDK>::LogTag()) {}
```

Comment:
Ideally, we can just run `LD_LOG_LEVEL=debug ./binary` but the Console Logger constructed in the client passes in a default value, so the environment variable isn't inspected.

Now it works like you'd want.